packages/create-redwood-app/src/create-redwood-app.js

```diff
@@ -621,44 +652,6 @@ async function createRedwoodApp() {
         'Get up and running fast with this Quick Start guide'
       )}: https://redwoodjs.com/docs/quick-start`,
```

Comment:
What do you think about making this `https://redwoodjs.com/quick-start` and adding a redirect? I can take care of that if you want.

Reply:
Makes sense. Happy to change it here and just confirm the redirect's going before release

Reply:
I'll add it now ✅

Reply:
Done in https://github.com/redwoodjs/redwood/pull/8261/commits/e4c9f254f4339e641f882007341ba773dce1e9ae